module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,16 @@
+#include "sort.hpp"
+
+#include <algorithm>
+
+std::deque<std::string> lengthSort(const std::forward_list<std::string>& listToSort) {
+    std::deque<std::string> output{begin(listToSort), end(listToSort)};
```

Comment:
hey :)
Can you tell me what does it mean? Why after output in {} you call begin and end of listToSort?

Reply:
This way deque is constructed based on the elements from list, so elements from list are simply copied to deque.